docusaurus.config.js

```diff
@@ -238,6 +238,12 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
         theme: require('prism-react-renderer/themes/dracula'),
         darkTheme: darkCodeTheme,
       },
+      algolia: {
+        appId: 'YUW9QEL53E',
```

Comment:
@Chever-John
Did you use your private email to register the appId and key on algolia?

Reply:
This is true as far as the facts are concerned. But I think I got access to Alex before I operated. I forwarded the email to Alex.

Reply:
Do we have an expiry date for the appId and key? if it works for a long time, I think the private email is not a big deal.

Reply:
Hi, I am back for the newest status.
1. I have consulted with the relevant developers at Apache APISIX and Apache Skywalking. Based on their experience, they believe that it is perfectly fine to register using a private account, and then if you need to replace it in the future, email the official.
2. I didn't get a valid answer about the key's expiration date, and I'm waiting for an update. It seems that everyone doesn't care about it.
3. I am waiting for the official's reply.